index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styletony.css">
```

Comment:
- [ ] Please I suggest you properly link your CSS file to the HTML file. 
In this case, your CSS file is `style.css` but in the link, it is `styletony.css`. 

```suggestion
    <link rel="stylesheet" href="style.css">
```